src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -5900,7 +5902,10 @@ GetLineByOffset(
         }
 
         IlOffset = symInfo.Offset + MethodOffs;
-
+#else
+        const String &moduleName = ModuleNameFromIP(TO_CDADDR(Offset));
+        return SymbolReader::GetLineByILOffset(moduleName.c_str(), MethodToken, MethodOffs, pLinenum, lpszFileName, cbFileName);
```

Comment:
Hopefully the FEEFEE sequence points are“filtered” out on the managed side. I don’t remember if it was though.